### ISSUE TITLE
Add warning at abapGit startup when edge browser control is used

### ISSUE THREAD

**Issue by christianguenter2** [abapGit/abapGit#6257]
Would ease root cause analysis for problems like #6253
https://docs.abapgit.org/guide-sapgui.html#sap-gui-for-windows

I already tried `cl_gui_frontend_services=>get_gui_properties` but it doesn't reveal that information. Maybe there's another way.

https://github.com/christianguenter2/abapGit/tree/debug-sapgui-properties

**Comment by christianguenter2** [comment]
If there's a way to detect it programmatically we can also issue a warning at startup.

**Comment by christianguenter2** [comment]
Seems like it's encoded in the browser string:
![grafik](https://user-images.githubusercontent.com/17437789/236391975-ab4d78b2-f3fc-41a8-ba80-caf3b917efcf.png)

vs with IE control
![grafik](https://user-images.githubusercontent.com/17437789/236392002-e090993c-a69e-4a1c-a94f-671e21a3ce03.png)

**Comment by mbtools** [comment]
Funny, i was working on this yesterday. the browser string is embedded into debug info using JS. 

There's `get_browser_string` for the HTML control. The method returns "MSIE_9.x.y" for IE. Apparently no one has used this method with Chrome control. It completely crashes SAPGUI (I tried 7.70 p 10 and 8.0 p 2). I opened a message... sorry, "case" ... with SAP. Even if we get a fix, we can't use it in abapGit.
